subprojects/docs/src/docs/userguide/javaPlugin.adoc

```diff
@@ -36,17 +36,23 @@ To use the Java plugin, include the following in your build script:
 The Java plugin assumes the project layout shown below. None of these directories need to exist or have anything in them. The Java plugin will compile whatever it finds, and handles anything which is missing.
 
 [[javalayout]]
-.Java plugin - default project layout
-[cols="a,a,a", options="header"]
-|===
-| Directory
-|
-| Meaning
+`src/main/java`::
+Production Java source.
 
-include::javaProjectMainLayout.adoc[]
-include::javaProjectTestLayout.adoc[]
-include::javaProjectGenericLayout.adoc[]
```

Comment:
Good catch. I hadn't realised that was why Asciidoctor inclusions were used here. It may make sense to update included files and revert the above change.